resources/function_help/json/BETWEEN

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "BETWEEN",
+  "type": "operator",
+  "groups": ["Operators"],
+  "description": "Returns 1 if value is within the specified range. To test for exclusion `NOT BETWEEN` can be used.",
+  "arguments": [{
+    "arg": "lower_bound AND higher_bound",
+    "description": "range bounds"
+  }],
+  "examples": [{
+    "expression": "'B' BETWEEN 'A' AND 'C'",
+    "returns": "1"
```

Comment:
Shouldn't these examples be returning true/false? (Not 0/1)

Reply:
I believe that's just a historic remnant because those operators were introduced before we had proper Boolean support in fields/expressions

Reply:
Should I change them all?

Reply:
Maybe better as a follow up PR...

Reply:
Yeah, because we should backport for the existing functions